eo-maven-plugin/src/test/java/org/eolang/maven/ContainsFiles.java

```diff
@@ -50,7 +50,7 @@ final class ContainsFiles extends TypeSafeMatcher<Path> {
      * Ctor.
      * @param glbs Patterns
      */
-    ContainsFiles(final String... glbs) {
+    public ContainsFiles(final String... glbs) {
```

Comment:
@Suban05 when you move the `AssembleMojoIT` to `org/eolang/maven` package - this change become unnecessary